### DISCUSSION TITLE
Sharing Glossaries

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#11728] [Ideas] [answered]
I am lucky to be administering several related projects, and one thing I love is to share glossaries between the projects.

In the Internet Privacy space we have generated a lot of new terminology, some by the Tor Project, some external.

So as I administer other projects like OnionShare, I can share our glossaries between projects. 

This is great!

## Proposal 1: Do not show external glossary terms if there is an entry on the project
One challenge we face, tho, is that we have a lot of duplicated definitions. I am interested on how OnionShare defines their own terminology, but if I already have an entry for, lets say, 'tor relays', on our glossary.... I don't need to know what they say on theirs.

It would be better if external entries are only shown when they are not present on the local glossary.

## Proposal 2: Allow to share glossaries without requiring admin access

I would like more projects to use our glossary, but I dont want to give admin permissions to random people on the Internet.
Is there another way to allow the use of our glossary?

I want to have the glossary of F-droid in Tor Project, in case we mention apps they distribute.  But I dont date to ask admin access just for that.

## Proposal 3: Discourage the translation of the Glossary explanation, or maybe do not show the English explanation if there is text on the language. 

When translators translate the explanation of a term, the glossary interface gets cluttered with the versions in english/local language of all the strings. i dont need to see both, is ok with only my language.
I can easily see the English if I really need to.

**Comment by nijel** [comment, marked as answer]
1. I think it is still useful to show both of them. If they diverge, it might actually be the motivation to review that.
2. The intention here is to avoid “stealing” or “injecting” glossaries, so you have to control both sides for that.
3. Two explanations can be tricky, see https://github.com/WeblateOrg/weblate/issues/11810 or https://github.com/WeblateOrg/weblate/issues/9082
